src/Sylius/Bundle/WebBundle/Behat/WebContext.php

```diff
@@ -46,7 +46,7 @@ public function iShouldBeOnTheLoginPage()
     }
 
     /**
-     * @Given /^go to "([^""]*)" tab$/
+     * @Given /^(?:|I )go to "([^""]*)" tab$/
```

Comment:
Well, I was thinking about the step ``When I go to ... tab``, for me it's much more natural than ``When go to ... tab``.